code/style.css

```diff
@@ -1,3 +1,197 @@
+:root {
+  --primary: #d9eaf1;
+  --secondary: #0b293f;
+  --third: #459ec6;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  font-family: "Montserrat", sans-serif;
+  background-color: var(--primary);
+  height: 100%;
+  width: 100%;
+  margin: 0;
+}
+
+.hero-image {
+  background: linear-gradient(rgba(0, 0, 0, 0.342), rgba(0, 0, 0, 0.774)),
+    url("./Images/GitHub-headpic.jpeg");
+  height: 50vh;
```

Comment:
I am curious what is the benefit of using vh units instead of px or em! Any reasons you preferred this?